test/integration/hotrestart_test.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+if [[ -f "test/integration/hotrestart_disabled" ]]; then
```

Comment:
@htuch @dnoe is there a better way to just exclude this test entirely if hot restart is not being via bazel define?